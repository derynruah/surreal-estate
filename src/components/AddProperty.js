import React from "react";

const AddProperty = () => <div><h3>Add Property Page</h3></div>;

export default AddProperty;