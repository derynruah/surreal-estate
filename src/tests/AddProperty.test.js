import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import { render } from "@testing-library/react";
import AddProperty from "../components/AddProperty";

const RouterRender = () => {
    return (
        <Router>
            <AddProperty />
        </Router>
    );
};

describe('AddProperty', () => {
    test('renders correctly', () => {
        const { asFragment } = render(<RouterRender />);

        expect(asFragment()).toMatchSnapshot();
    });
});