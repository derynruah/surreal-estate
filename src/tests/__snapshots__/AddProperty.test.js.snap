// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddProperty renders correctly 1`] = `
<DocumentFragment>
  <div>
    <h3>
      Add Property Page
    </h3>
  </div>
</DocumentFragment>
`;
